src/store/actions/cohorts.actions.js

```diff
@@ -13,3 +13,22 @@ export const fetchCohortsThunk = () => dispatch => {
     .get(API_URL)
     .then(res => dispatch(setCohortsData(res.data.data)));
 };
+
+export const fetchActiveCohort = cohortID => dispatch => {
+  //TODO: add error handling
+  dispatch({ type: SINGLE_COHORT.GET_COHORT_PENDING });
+
+  return axios
+    .get('http://www.mocky.io/v2/5de32f5a3000007200e9c988')
```

Comment:
The API call should be done using the cohort id and therefore return the data for a single cohort.
e.g ``` .get('....url/cohort-Id') ``` then receive an object containing aggregate cohort data such as: 
```
{
  cohort: "cohort-6",
  gender: [],
   workStatus: []
}
```
so no need to get a cohort from an array and it already contains the cohort id to be used.

Reply:
@IrinaGM I am not sure I understand this comment. It doesn't agree with what @vindhya said few comments above. I think we should agree on one api call response. Is it 
```{
data: {
    "gender": [
      { "name": "female", "count": 40 },
      { "name": "male", "count": 4 }
    ], 
   "minorityGroup": [ ... ],
   ...
  }
}```
or 
```{
  cohort: "cohort-6",
  gender: [],
   workStatus: []
}```